### PR TITLE
Feature/login POST Route

### LINKED ISSUE #1: Add endpoint `POST /login`
- Request:
    - `id` - ID Number
    - `password` - Password
  - Response Data:
    - `token` - JWT Token

### PR DESCRIPTION
**Pull request by TotalElderBerry**
closes #1

### CODE REVIEW

File: src/core/auth/login.ts [hunk 1]
```diff
@@ -1,35 +1,54 @@
...
  */
-
 import type {Response, Request} from "express"
 import { result } from "../../utils/response";
 import jwt from 'jsonwebtoken';
+import Student from "../../db/models/student"
+import bcrypt from 'bcrypt';
 
 
 export function login(request: Request, response: Response){
```
Comment (mavyfaby) [inline src/core/auth/login.ts:11]: Add this `login` function in routes `src/routes/index.ts`

File: src/core/auth/login.ts [hunk 2]
```diff
@@ -1,35 +1,54 @@
...
     switch(request.method){
         case 'GET':
```
Comment (mavyfaby) [inline src/core/auth/login.ts]: You can remove `GET` case here since only `POST` request will be used.

Comment (TotalElderBerry) [reply src/core/auth/login.ts]: noted with thanks sir. have a nice day!

**Comment by mavyfaby** [comment]
LGTM

**Pull request merged**